app/Region/Standard/DarkWorld/NorthEast.php

```diff
@@ -221,6 +221,12 @@ public function initalize()
                 return false;
             }
 
+            if ($this->world->config('goal') == 'completionist'
+                && $this->world->config('accessibility') !== 'locations'
+            ) {
+                return false;
+            }
```

Comment:
this is odd feeling, this would be it's configured to fail? feels like it could be handled a little earlier in the whole thing, but it's not that big of a deal, this also looks to get the job done.

Reply:
I agree, moved it to the beginning of the prepareWorld function in Randomizer.php so it will fail before it does much of anything.